cupy/testing/helper.py

```diff
@@ -8,6 +8,7 @@
 from unittest import mock
 import warnings
 
+import _pytest
```

Comment:
`_pytest.outcomes` is not available unless there is `import pytest`.
```suggestion
import _pytest.outcomes
```

Reply:
`import cupy`, which makes `cupy.testing.helper` available for some reason, should not fail with newer PyTest (than the version of the test requirement: `pytest<4.2`).  It would be safer either
- `import _pytest.outcomes` inside `for_dtypes`, or
- `try: import _pytest.outcomes except: pass`.

However it does not seem an immediate issue, because I confirmed that `_pytest.outcomes` is still available in `pytest==6.1.1`.

Reply:
Sorry.  First of all, `pytest` does not have to be installed.  Could you follow `cupy.testing.attr`?
https://github.com/cupy/cupy/blob/0a8b1104aae25ee5973c126f3634b54a1bf9622e/cupy/testing/attr.py#L5-L21

Reply:
I fixed as such. Would you take a look again?